README.md

```diff
@@ -1,2 +1,73 @@
-# prometheus-opensearch-dashboards-exporter
+# Prometheus Opensearch Dashboards Exporter
 An exporter for OpenSearch Dashboards
+
+
+## Installing
+With [poetry](https://python-poetry.org/) installed, clone this repo and run:
+
+```shell
+poetry install
```

Comment:
This gives me the following error right now:

```
paul-goins@pdg-thinkpad:~/code/prometheus-opensearch-dashboards-exporter$ poetry install

The Poetry configuration is invalid:
  - The fields ['authors', 'description', 'name', 'version'] are required in package mode.
```

Reply:
Are you using the Poetry version 2.0? I've updated to use the latest release and there are some breaking changes if you are using 1.X

Reply:
This needs to be documented in the README, especially since the python3-poetry package in even the latest released ubuntu is still on 1.8.3.  I might even suggest reverting back to 1.8.2 so we can at least retain compatibility with the latest Ubuntu LTS, otherwise we're basically requiring Ubuntu users to set up a virtualenv (or otherwise perform a local install of Poetry outside of their package manager) in order to run Poetry in these environments.

If you have a strong reason for wanting Poetry 2 instead of 1.8.2, that's fine, but I think the README needs to be updated to make this requirement clear.  Otherwise, I'd suggest rolling back.

Reply:
I've updated the README to include that poetry needs to be >=2.0.0

I don't have a strong reason,  but I think it's an issue having the newest version. [Charmed Opensearch](https://github.com/canonical/opensearch-operator/blob/2/edge/pyproject.toml#L6) itself is already using Poetry > 2.0 because they found some [CI issues](https://github.com/canonical/opensearch-operator/pull/527), so I decided to go in the same path